Source/WebKit/UIProcess/API/Cocoa/WKPreferences.mm

```diff
@@ -187,14 +187,17 @@ - (void)setInactiveSchedulingPolicy:(WKInactiveSchedulingPolicy)policy
     case WKInactiveSchedulingPolicySuspend:
         _preferences->setShouldTakeNearSuspendedAssertions(false);
         _preferences->setBackgroundWebContentRunningBoardThrottlingEnabled(true);
+        _preferences->setShouldDropNearSuspendedAssertionAfterDelay(WebKit::defaultShouldDropNearSuspendedAssertionAfterDelay());
         break;
     case WKInactiveSchedulingPolicyThrottle:
         _preferences->setShouldTakeNearSuspendedAssertions(true);
         _preferences->setBackgroundWebContentRunningBoardThrottlingEnabled(true);
+        _preferences->setShouldDropNearSuspendedAssertionAfterDelay(false);
         break;
     case WKInactiveSchedulingPolicyNone:
         _preferences->setShouldTakeNearSuspendedAssertions(true);
         _preferences->setBackgroundWebContentRunningBoardThrottlingEnabled(false);
```

Comment:
This has no effect on iOS. I wonder if we should document that `WKInactiveSchedulingPolicyNone` is the same as `WKInactiveSchedulingPolicyNone` on that platform.